src/go/k8s/tests/e2e/admin-api-tls/01-call-adminapi.yaml

```diff
@@ -23,7 +23,7 @@ spec:
             - -c
             - -ex
           args:
-            - curl https://cluster-tls-0.cluster-tls.$POD_NAMESPACE.svc.cluster.local:9644/metrics -v --cacert /etc/tls/certs/admin/ca.crt
+            - curl https://cluster-tls-0.cluster-tls.$POD_NAMESPACE.svc.cluster.local:9645/metrics -v --cacert /etc/tls/certs/admin/ca.crt
```

Comment:
ditto

Reply:
Only the external port has TLS enabled.

Reply:
I modified the code to avoid this requirement.